.travis.yml

```diff
@@ -1,9 +1,24 @@
-language: node_js
-node_js:
-  - "6"
+language: java
+jdk:
+- oraclejdk8
+before_install:
+- java -version
+- wget https://raw.githubusercontent.com/creationix/nvm/v0.31.0/nvm.sh -O ~/.nvm/nvm.sh
+- source ~/.nvm/nvm.sh
+- nvm install 6.9.1
+- node --version
+- npm install
 services:
 - postgresql
 addons:
   postgresql: "9.4"
+before_script:
+- npm run test
+- npm run example &
+script:
+- source ./scripts/OGC_tests/install_TE.sh
```

Comment:
Could we unify these scripts into a single one? I think it'll be easier to keep track of the required step this way and it'll help understand better all the dir changes that the scripts are doing.